doc/ref_cert/RC1/chapters/chapter02.md

```diff
@@ -293,6 +294,7 @@ Conformance and issuance of NFVI badges will be as follows:
 ![NFVI Badges](../figures/NFVI_certifying_vendor_swhw_solutions.jpg)
```

Comment:
@colivier You will need to add the Figure caption as in:
![NFVI Badges](../figures/NFVI_certifying_vendor_swhw_solutions.jpg) **NFVI Badges**

Reply:
@pgoyal01 then I do check how it would be converted into RST. Yes I take the action. Thx